src/runtime/security/access_controller.cpp

```diff
@@ -40,6 +51,14 @@ bool access_controller::pre_check(const std::string &user_name)
         return true;
     }
     return false;
+    return _super_users.find(user_name) != _super_users.end();
+}
+
+bool access_controller::is_enable_ranger_acl() { return FLAGS_enable_ranger_acl; }
```

Comment:
Why not use `FLAGS_enable_ranger_acl` directly?

Reply:
Both `dsn_meta_server` and `dsn_replica_server` will use it, so `FLAGS_enable_ranger_acl` cannot be used directly.

Reply:
yes